src/gurt/telemetry.c

```diff
@@ -763,65 +778,69 @@ d_tm_compute_standard_dev(double sum_of_squares, uint64_t sample_size,
 }
 
 /**
- * Compute gauge statistics: sample size, min, max, sum and sum of squares.
+ * Compute statistics: sample size, min, max, sum and sum of squares.
  * Standard deviation calculation is deferred until the metric is read.
  *
- * \param[in]	node		Pointer to a gauge node
+ * \param[in]	node		Pointer to a node with stats
  */
 void
-d_tm_compute_gauge_stats(struct d_tm_node_t *node)
+d_tm_compute_stats(struct d_tm_node_t *node, uint64_t value)
```

Comment:
value needs to be added to the doxygen comment above

Reply:
fixed